packages/dm-core-plugins/src/explorer/components/dialogs/NewEntityDialog.tsx

```diff
@@ -22,13 +25,15 @@ type TProps = {
 
 const NewEntityDialog = (props: TProps) => {
   const { setDialogId, node, setNodeOpen } = props
-  const [blueprint, setBlueprint] = useState<string>('')
+  const [blueprintName, setBlueprintName] = useState<string>('')
+  const [newName, setNewName] = useState<string>(defaultName)
   const [loading, setLoading] = useState<boolean>(false)
+  const { blueprint } = useBlueprint(blueprintName)
```

Comment:
This will lead to some 404 before a blueprint is selected? Move it to a useEffect or something?